doc/source/user/usage.rst

```diff
@@ -194,6 +194,20 @@ the plugin are forwarded to the :func:`openmc.run` function. For example::
 
     results = openmc_plugin(params, mpi_args=["mpiexec", "-n", "16"])
 
+By default, the OpenMC plugin will only call the :func:`openmc.run` function,
+but you can customize the execution by passing an arbitrary function as the
+``function`` keyword argument. For example, if you wanted to additionally call
+:func:`openmc.plot_geometry` each time the plugin is called, this could be
+accomplished as follows::
+
+    import openmc
+
+    def run_function():
+        openmc.plot_geometry()
+        openmc.run()
+
+    results = openmc_plugin(params, function=run_function)
```

Comment:
this looks great - I would suggest to provide in the documentation an extensive list of openmc functions that it would make sense to call. For instance `calculate_volume()`, etc.

Reply:
Good suggestion -- I've added a "See also" section in the docstring for `PluginOpenMC.run` that lists common functions